lib/fastlane/plugin/wpmreleasetoolkit/models/version.rb

```diff
@@ -0,0 +1,312 @@
+module Fastlane
+  module Helper
+    RC_DELIMITERS = %w[
+      rc
+      beta
+      b
+    ].freeze
+
+    class Version
+      include Comparable
+      attr :major, :minor, :patch, :rc
+
+      def initialize(major:, minor:, patch: 0, rc_number: nil)
+        @major = major
+        @minor = minor
+        @patch = patch
+        @rc = rc_number
+      end
+
+      # Create a new Version object based on a given string.
+      #
+      # Can parse a variety of different two, three, and four-segment version numbers,
+      # including:
+      # - x.y
+      # - x.yrc1
+      # - x.y.rc1
+      # - x.y-rc1
+      # - x.y.rc.1
+      # - x.y-rc-1
+      # - x.y.z
+      # - x.y.zrc1
+      # - x.y.z.rc1
+      # - x.y.z-rc1
+      # - x.y.z.rc.1
+      # - x.y.z-rc-1
+      # - Any of the above with `v` prepended
+      #
+      def self.create(string)
+        string = string.downcase
+        string = string.delete_prefix('v') if string.start_with?('v')
```

Comment:
`delete_prefix` already checks if the string starts with said prefix (and is a no-op if it doesn't)

```suggestion
        string = string.delete_prefix('v')
```